tests/unit/backend/wmg/fixtures/test_snapshot.py

```diff
@@ -208,7 +212,7 @@ def load_realistic_test_snapshot(snapshot_name: str) -> WmgSnapshot:
                 marker_genes_cube=marker_genes_cube,
                 cell_counts_cube=cell_counts_cube,
                 cell_type_orderings=None,
-                primary_filter_dimensions=None,
+                primary_filter_dimensions=primary_filter_dimensions,
```

Comment:
setting primary filter dimensions in the test snapshot. previously, it was left to None as it was not needed.